README.md

```diff
@@ -106,6 +106,13 @@ bower install --save most`
 <script src="most/dist/most.js"></script>
 ```
 
+### Typescript support
+
+Most.js works with typescript out of the box as it provides local typings that will be read when you import Most.js in your code. You do not need to manually link an external `d.ts` file in your tsconfig.  
```

Comment:
How about linking "provides local typings" to https://github.com/cujojs/most/blob/master/type-definitions/most.d.ts